integration-tests/README.md

```diff
@@ -1,36 +1,67 @@
-An integration test harness that runs RN apps integrated with the Embrace SDK and sets up a mock server to allow
-verifications on the payloads that would have been sent to Embrace.
+The tooling in this directory is intended to facilitate both manual QA and integration testing of React Native apps
+integrated with the Embrace SDK.
 
 ## Setup
 
-The harness uses WebdriverIO's [Testrunner](https://webdriver.io/docs/testrunner/) to spin up a [Appium](http://appium.io/docs/en/latest/intro/) client and server to perform the device automation. [Mockserver](https://www.mock-server.com/#what-is-mockserver) is
-launched for the suite run so that requests sent from the device can be inspected.
-
 ```bash
 npm install
 ```
 
-For any future updates the [Appium Installer](https://webdriver.io/docs/appium) provides a handy setup wizard:
+## Test Applications
+
+Because React Native has not yet reached 1.0.0 each minor version release has the potential to introduce breaking
+changes, so we aim to test the Embrace SDK across a range of current and past minor versions
+([see here](https://reactnative.dev/versions) for a current list).
+
+To help spin up these apps we leverage react native templates and include one per minor version / framework we want to
+do testing on.
+
+### Creating new application templates
+
+A simple way to create a new template is to start by creating an app using `@react-native-community/cli init` at the
+desired RN version:
 
 ```bash
-npx appium-installer
+npx @react-native-community/cli init ProjectName --skip-git-init --skip-install --pm yarn --version 0.x.y
 ```
 
-## Create test apps
+Then moving over the created app into the templates folder (`mv ProjectName templates/my-new-template`), removing any
+unneeded files, and adding Embrace specific setup.
+
+The ["Current Tags"](https://www.npmjs.com/package/react-native?activeTab=versions) section of the react-native package
+in NPM can help decide which specific patch version to pin the template to for a given minor version, there will generally
+be one that has been tagged with `0.minor-stable`.
 
-New test apps can be created from templates as needed. To create a new test expo app run:
+A similar approach can be done using Expo's cli for creating new apps:
 
 ```bash
-npx create-expo --template ./templates/<template-app>/<artifact>.tgz
+npx create-expo
 ```
 
-To create a new bare react native app run:
+Though note that in this case a version can't be passed as different releases of expo are tied to particular RN versions,
+instead you can run a different version of the `create-expo` package.
+
+### Creating new apps from templates
+
+New test apps can be created from templates as needed. To create a new bare react native app run:
 
 ```bash
 npx @react-native-community/cli init <test-app> --package-name io.embrace.<test-app> --skip-git-init --skip-install --pm yarn --template $(pwd)/templates/react-native-test-app-template
```

Comment:
suggesting `<testApp>` instead of `<test-app>` since when I tried to create an app using dashes the CLI threw an error saying that it wasn't a valid name (it only accepted alphanumeric characters)